### PR TITLE
[FREELDR] Separate PC beep routines out into it's own file

### PR DESCRIPTION
**Pull request by disean**
## Purpose

i386rtl.c doesn't contains RTL functions nowadays.

### CODE REVIEW

File: boot/freeldr/freeldr/arch/i386/pc/pcbeep.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    WRITE_PORT_UCHAR((PUCHAR)0x43, 0xB6);
+    WRITE_PORT_UCHAR((PUCHAR)0x42, Scale & 0xFF);
+    WRITE_PORT_UCHAR((PUCHAR)0x42, Scale >> 8);
+    WRITE_PORT_UCHAR((PUCHAR)0x61, READ_PORT_UCHAR((PUCHAR)0x61) | 3);
+}
+
+VOID PcBeep(VOID)
+{
+    Sound(700);
+    StallExecutionProcessor(100000);
```
Comment (HBelusca) [inline boot/freeldr/freeldr/arch/i386/pc/pcbeep.c]: Why not using 200 milliseconds as was done in the original code?

File: boot/freeldr/freeldr/arch/i386/i386rtl.c [hunk 2]
```diff
@@ -1,117 +0,0 @@
...
-void delay(unsigned msec);
-
-VOID PcBeep(VOID)
-{
-    sound(700);
-    delay(200);
-    sound(0);
-}
-
-void delay(unsigned msec)
```
Comment (HBelusca) [inline boot/freeldr/freeldr/arch/i386/i386rtl.c]: It's good that you remove that function and instead use the more accurate StallExecutionProcessor().
I would then suggest you to go ahead and fix the implementation of the `KeStallExecutionProcessor()` stub at https://github.com/reactos/reactos/blob/master/boot/freeldr/freeldr/arch/i386/ntoskrnl.c#L99 by just calling the `StallExecutionProcessor()` within (the parameters are the same and are in microseconds), and removing the pcbios thing at the top of this same file https://github.com/reactos/reactos/blob/master/boot/freeldr/freeldr/arch/i386/ntoskrnl.c#L13 .

File: boot/freeldr/freeldr/arch/i386/pc/pcbeep.c (hunk 1 above)
Comment (disean) [reply boot/freeldr/freeldr/arch/i386/pc/pcbeep.c]: The changes is improve the sound quality if the keyboard key isn't released.

File: boot/freeldr/freeldr/arch/i386/i386rtl.c (hunk 2 above)
Comment (disean) [reply boot/freeldr/freeldr/arch/i386/i386rtl.c]: Improved.

**Pull request merged**